activesupport/test/core_ext/range_ext_test.rb

```diff
@@ -206,11 +177,6 @@ def test_step_on_time_with_zone
     end
   end
 
-  def test_cover_on_time_with_zone
-    twz = ActiveSupport::TimeWithZone.new(nil, ActiveSupport::TimeZone["Eastern Time (US & Canada)"], Time.utc(2006, 11, 28, 10, 30))
-    assert ((twz - 1.hour)..twz).cover?(twz)
```

Comment:
Don't remove the test `cover?` on time with zone.

Reply:
Sure! 😄